setup.cfg

```diff
@@ -0,0 +1,2 @@
+[pydocstyle]
+select = D1
```

Comment:
@perrygeo @dnomadb I think that starting at D1xx and increasing incrementally could be a good pattern.